ee/ucp/admin/configure/configure-rbac-kube.md

```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
 
-Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all the roles for a particular cluster under **Access Control** then **Roles**. Select Kubernetes or Swarm to view the specific roles for each.
+Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all of the roles for a particular cluster under **Access Control** > **Roles**. Select **Kubernetes** or **Swarm** to view the specific roles for each.
 
 ## Creating roles
 
-You create Kubernetes roles either through the CLI using `kubectl` or through the UCP web interface.
-
-To create a Kubernetes role in the UCP web interface:
+You can create Kubernetes roles either through the CLI using [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) or through the UCP web interface. The following procedure uses the UCP web interface.
 
 1. From the UCP UI, select **Access Control**.
 2. From the left navigation menu, select **Roles**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32roles.png)
 3. Select the **Kubernetes** tab at the top of the window.
 4. Select **Create** to create a Kubernetes role object in the following dialog:
     ![Kubernetes Role Creation in UCP](/ee/ucp/images/kube-role-create.png)
-5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a `ClusterRole` where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
+5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a ClusterRole where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
 6. Provide the YAML for the role, either by entering it in the **Object YAML** editor or select **Click to upload a .yml file** to choose and upload a .yml file instead.
-7. When you have finished specifying the YAML, Select **Create** to complete role creation.
+7. Select **Create**.
 
 ## Creating role grants
 
-Kubernetes provides 2 types of role grants:
+Kubernetes provides two types of role grants, **ClusterRoleBinding** and **RoleBinding**:
 
-- `ClusterRoleBinding` which applies to all namespaces
-- `RoleBinding` which applies to a specific namespace
+- **ClusterRoleBinding** which applies to all namespaces
+- **RoleBinding** which applies to a specific namespace
 
 To create a grant for a Kubernetes role in the UCP web interface:
 
 1. From the UCP UI, select **Access Control**.
 2. From the left navigation menu, select  **Grants**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32grants.png)
 3. Select the **Kubernetes** tab at the top of the window. All grants to Kubernetes roles can be viewed in the Kubernetes tab.
```

Comment:
Edit into one sentence, "Select the **Kubernetes** tab at the top of the window to view all of the grants available to Kubernetes roles."